code/modules/shuttle/custom_shuttle/shuttle_creator_overlay.dm

```diff
@@ -0,0 +1,52 @@
+/*
+ * Manages the overlays for the shuttle creator drone.
+*/
+
+/datum/shuttle_creator_overlay_holder
+	var/client/holder
+	var/list/images = list()
+	var/list/turfs = list()
```

Comment:
if each turf has an image, just do
```
var/list/turfs_to_image
```
as an assoc list.
will make the "are we already tracked" check better too